### ISSUE TITLE
BCFViewpointsPlugin - getViewpoint spaces_visible issues

### ISSUE THREAD

**Issue by xeolabs** [xeokit/xeokit-sdk#285]
If I capture a viewpoint with 

Using ````BCFViewpointsPlugin````, if I capture a viewpoint with the ````spaces_visible```` option:

````javascript
const vp = myPlugin.getviewpoint({
    spaces_visible: true
});
````

where only some ````IfcSpace```` elements are visible, then restore again:

````javascript
myPlugin.setviewpoint(vp);
````

then **all the ````IfcSpace```` elements are made visible again**, whereas I might just want to show only the ````IfcSpaces```` that were visible when I saved the viewpoint.

I want to change behaviour so that only the ````IfcSpace```` elements that were previously visible become visible again.

@Amoki would this change cause problems for BIMData?

**Comment by Amoki** [comment]
It shouldn't cause problems, and if it does we'll update our code!

**Comment by Amoki** [comment]
I just noticed that spaces are also pickable again if they were unpikable before the `setViewpoint()`

**Comment by xeolabs** [comment]
@amoki that pickable behaviour is actually part of the way ````reset```` works for setViewpoint(): https://github.com/xeokit/xeokit-sdk/commit/2bcfb9a3e30ba0380e0519b4ef22c68c79e77512#diff-ee1102a6f42113c9ee1d51304502ad77

I've just removed the logic that makes all objects pickable and opaque as part of setViewpoint with ````reset````, since those states are not modified by BCF viewpoints. 

@Kurtil just a heads-up that I removed that state (see above comments). In case you're depending on it, to reset those pickable and opacity states, try using the ````[ObjectMemento](https://xeokit.github.io/xeokit-sdk/docs/class/src/viewer/scene/mementos/ObjectsMemento.js~ObjectsMemento.html)```` class.
